tools/CouchDB_Import_MRI.php

```diff
@@ -199,6 +229,110 @@ function _addMRIFeedback($current_feedback, $scan_type, $mri_feedback)
     }
 
     /**
+    * Gets the date according to the type
+    *
+    * @param MRIFile $file  file object
+    * @param string  $type  type of the date
+    * @param array   $array array containing the date
+    *
+    * @return date if exists, if not an empty string
+    */
+    function _getDate($file, $type, $array)
+    {
+        if (preg_match(
+            "/(\d{4})-?(\d{2})-?(\d{2})/", 
+            $file->getParameter($type), $array
+        )) {
+            return (mktime(12, 0, 0, $array[2], $array[3], $array[1]));
+        } else {
+            return "";
+        }
+    }
+
+    /**
+    * Gets a rejected parameter according to its type
+    *
+    * @param MRIFile $file  file object
+    * @param string  $type  type of the rejected
+    * @param array   $array array containing rejected
+    *
+    * @return parameter of the rejected
```

Comment:
What's a "parameter of the rejected"? The parameter value from the parameter type table? From the minc header? The ID of the ParameterTypeID? Etc.. this @return parameter doc isn't clear.